android/src/main/java/tv/vizbee/rnsender/VizbeeCastButtonView.java

```diff
@@ -21,6 +21,10 @@ public VizbeeCastButtonView(ThemedReactContext themedReactContext) {
         this.addView(m_button);
     }
 
+    public void setColor(String color){
+        Log.d(TAG, "setColor " + color + " for view "+ m_button.getId());
```

Comment:
Let's follow the standard logging approach

1- Which is, LOG_TAG should have "VZBRNSDK_<className> for easy filtering
`private static final String LOG_TAG = "VZBRNSDK_VizbeeNativeManager";`

2 - Log.v(LOG_TAG, "setTintColor - " + tintColor);